lib/README.md

```diff
@@ -295,3 +295,110 @@ if (false === $result) {
 ```
 
 This function returns the error message if any.
+
+### Class `WasmArrayBuffer`
+
+This class represents a buffer of bytes. It will be used to manipulate
```

Comment:
This documentation lacks an usage example for these two: from a reader's perspective, you described an implementation, but not the API

Reply:
Yeah, that's because it's not plugged to the Wasm memory API yet. I'll update that part after.

It may look like this:

```php
$instance = new Wasm\Instance(…);
$memory_as_uint8 = new WasmUint8Array($instance->getMemoryBuffer());
```